apigw-lambda-s3-terraform-java/README.md

```diff
@@ -0,0 +1,107 @@
+# Upload file up to 5GB to Amazon S3 bucket using presigned URL using Terraform 
```

Comment:
I understand the intention but I suggest moving the 5 GB limitation to the paragraph below instead of the title

Reply:
Got it. I have moved the 5GB limitation to the paragraph below!